### DISCUSSION TITLE
Update docs to reflect correct flag

### DISCUSSION THREAD

**Discussion by jake-luna-sada** [mondoohq/cnquery#2593] [General]
Hello, going through some of the documentation and testing things out I noticed there is a typo  here https://console.mondoo.com/space/integrations/add/mondoo/setup?spaceId=pedantic-kepler-689646

In particular `cnspec scan k8s --namespace` where is should be cnspec scan k8s --namespaces. Though personal opinion  it should be --namespace to match existing kubectl commands

**Comment by tas50** [comment]
Thanks for the report @Jluna-sada. I cut a PR to fix this.

**Comment by jake-luna-sada** [comment]
Wow fast turn around, thank you very much!

**Comment by tas50** [comment]
Fix is out in production now. Thanks @Jluna-sada
